docs/source/release/v3.14.0/reflectometry.rst

```diff
@@ -31,6 +31,8 @@ Improved
 - Common naming of slit component name and size properties across algorithms.
 - :ref:`algm-SpecularReflectionPositionCorrect` is now compatible with the reflectometers at ILL.
 - :ref:`algm-CreateTransmissionWorkspace` and :ref:`algm-CreateTransmissionWorkspaceAuto` now use NormalizeByIntegratedMontitors instead of using MonitorIntegrationWavelengthMin and MonitorIntegrationWavelengthMax being defined, to determine how to normalize. 
+- :ref:`algm-ReflectometryILLPreprocess` does not accept and will not output an `DirectBeamPosition` and `OutputBeamPositionWorkspace`, respectively. However, new input properties are present to refine the peak fitting. In general, D17 and Figaro instrument definition files are modified to let the Spectrum numbers begin with index 0 instead of 1. The peak position are now present in the SampleLog information as documented.
```

Comment:
I thought the loader numerates the spectrum numbers.

Reply:
Yes, I will re-write the release notes.